x/pairing/keeper/staking.go

```diff
@@ -143,6 +143,25 @@ func (k Keeper) StakeNewEntry(ctx sdk.Context, validator, creator, chainID strin
 		{Key: "geolocation", Value: geolocation},
 	}
 
+	// if there are registered delegations to the provider, count them in the delegateTotal
+	delegateTotal := sdk.ZeroInt()
+	epoch := k.epochStorageKeeper.GetEpochStart(ctx)
```

Comment:
use the next epoch instead of current one

Reply:
done. see https://github.com/lavanet/lava/pull/1188/commits/ed65639f3db931943b5b007ffea26726ce1b5e93